bluepyopt/ephys/parameterscalers.py

```diff
@@ -97,16 +97,17 @@ def scale(self, value, segment, sim=None):
         soma = segment.sec.cell().soma[0]
 
         # Initialise origin
-        sim.neuron.h.distance(0, 0.5, sec=soma)
+        sim.neuron.h.distance(0, 0, sec=soma)
```

Comment:
Is this what Neurodamus does ? 
Mmm, need to discuss this with M. Hines, should probably be .5